### PR TITLE
fix: updated api to 3.8.2

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**

### CODE REVIEW

File: Common/tests/Pollster/TaskHandlerTest.cs [hunk 1]
```diff
@@ -876,7 +876,7 @@ public void Dispose()
     {
     }
 
-    public IAsyncPipe<ProcessReply, ProcessRequest>? Pipe { get; private set; }
```
Comment (aneojgurhem) [inline Common/tests/Pollster/TaskHandlerTest.cs:879]: This should not be impacted

**Comment by aneojgurhem** [comment]
From which version did you start ?

File: Common/tests/Pollster/TaskHandlerTest.cs (hunk 1 above)
Comment (dbrasseur-aneo) [reply Common/tests/Pollster/TaskHandlerTest.cs:879]: Looks like Resharper decided to remove it during clean up

Comment (aneojgurhem) [reply Common/tests/Pollster/TaskHandlerTest.cs:879]: This should not be changed in this PR

Comment (dbrasseur-aneo) [reply Common/tests/Pollster/TaskHandlerTest.cs]: removed

**Review by aneojgurhem — APPROVED** [review]

**Review by lemaitre-aneo — DISMISSED** [review]

File: Tests/Common/Client/src/ArmoniK.Core.Common.Tests.Client.csproj [hunk 2]
```diff
@@ -23,9 +23,9 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
-
+
```
Comment (aneojgurhem) [inline Tests/Common/Client/src/ArmoniK.Core.Common.Tests.Client.csproj:26]: You introduced spaces !!

Comment (aneojgurhem) [reply Tests/Common/Client/src/ArmoniK.Core.Common.Tests.Client.csproj:26]: Can you remove them

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**